### PR TITLE
RESTXQ 2.0 Spec.

### PR DESCRIPTION
**Pull request by yamahito**
Creates stub 2.0 Spec

### CODE REVIEW

File: exquery-restxq-specification/restxq-2.0-specification.html [hunk 1]
```diff
@@ -0,0 +1,925 @@
...
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+    ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+    SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+-->
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RESTXQ 2.0: RESTful Annotations for XQuery</title>
```
Comment (adamretter) [inline exquery-restxq-specification/restxq-2.0-specification.html:33]: As discussed in the draft paper, this should just be: "RESTXQ 2.0: RESTful XQuery"

File: exquery-restxq-specification/restxq-2.0-specification.html [hunk 2]
```diff
@@ -0,0 +1,925 @@
...
+    <meta charset='utf-8'/>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
+    <script class='remove'>
+      var respecConfig = {
+          // specification status (e.g. WD, LCWD, NOTE, etc.). If in doubt use ED.
+          specStatus:           "unofficial",
+
+          // the specification's short name, as in http://www.w3.org/TR/short-name/
+          shortName:            "restxq",
```
Comment (adamretter) [inline exquery-restxq-specification/restxq-2.0-specification.html:42]: This should be restxq-20

File: exquery-restxq-specification/restxq-2.0-specification.html [hunk 3]
```diff
@@ -0,0 +1,925 @@
...
+          edDraftURI:           "http://exquery.github.com/exquery/exquery-restxq-specification/restxq-2.0-specification.html",
+
+          // if this is a LCWD, uncomment and set the end of its review period
+          // lcEnd: "2009-08-05",
+
+          // editors, add as many as you like
+          // only "name" is required
+          editors:  [
+              { name: "Adam Retter", url: "http://adamretter.org.uk/",
+                company: "Evolved Binary", companyURL: "http://adamretter.org.uk/" }
```
Comment (adamretter) [inline exquery-restxq-specification/restxq-2.0-specification.html:71]: companyURL should be `https://www.evolvedbinary.com` please

**Review by adamretter — CHANGES_REQUESTED** [review]

File: exquery-restxq-specification/restxq-2.0-specification.html (hunk 3 above)
Comment (yamahito) [reply exquery-restxq-specification/restxq-2.0-specification.html:71]: whoops, spotted that in the authors segment, but not editors.  Fixing...

**Review by adamretter — APPROVED** [review]
LGTM. Thanks @yamahito

**Pull request merged**